proposals/lip-0058.md

```diff
@@ -730,10 +728,7 @@ This function returns the slot number corresponding to the input timestamp.
 
 ```python
 def getSlotNumber(timestamp):
-    if timestamp < genesisTimestamp:
-        raise Exception('Input timestamp cannot be smaller than genesis block timestamp.')
-    elapsedTime = timestamp - genesisTimestamp
-    return floor(elapsedTime / BLOCK_TIME)
+    return floor(timestamp / BLOCK_TIME)
 ```
 
 ### Genesis Block Processing
```

Comment:
At the moment `genesisTimestamp` is still set during the genesis block processing. I would remove it from there.

Reply:
True, shouldn't be there. I removed it.